gh-pages/webdx-cg.md

```diff
@@ -0,0 +1,17 @@
+---
+permalink: webdx-cg
+---
+
+# The WebDX Community Group
+
+> The mission of the [Web Developer Experience (WebDX) Community Group](https://www.w3.org/community/webdx/) is to facilitate coordinated approaches to improve the overall experience of developing for the Web platform when such coordination provides unique opportunities for these improvements. — [WebDX Community Group charter](https://github.com/web-platform-dx/admin/blob/main/charter.md).
+
+A subset of the members of the CG make up the owner and governance body for Baseline. The official definition of what Baseline is, including the goals and non-goals of the project, can be found in the [web-platform-dx/web-features repository](https://github.com/web-platform-dx/web-features/blob/main/docs/baseline.md).
+
+The community group also owns the [web feature grouping work](web-features).
```

Comment:
I would actually simplify the intro even more:

```suggestion
The [Web Developer Experience (WebDX) Community Group](https://www.w3.org/community/webdx/) is a W3C Community Group which coordinates the work on [web-features](/web-features) and [Baseline](/baseline). For more information, see the group's [charter](https://github.com/web-platform-dx/admin/blob/main/charter.md).
```

Reply:
Going to stick with @ddbeck suggestion for now. Feel free to reopen if you feel strongly.